.github/workflows/scripts/release-changelog.php

```diff
@@ -14,21 +14,17 @@
 $release_time = strtotime( '+26 days', $now );
 $release_date = date( 'Y-m-d', $release_time );
 
-$changelog_file = $base_dir . '/changelog.txt';
 $readme_file    = $base_dir . '/plugins/woocommerce/readme.txt';
 $next_log_file  = $base_dir . '/plugins/woocommerce/NEXT_CHANGELOG.md';
 
-$changelog = file_get_contents( $changelog_file );
 $readme    = file_get_contents( $readme_file );
 $next_log  = file_get_contents( $next_log_file );
 
 $next_log  = preg_replace( "/= (\d+\.\d+\.\d+) YYYY-mm-dd =/", "= \\1 {$release_date} =", $next_log );
 
 // Convert PR number to markdown link.
 $next_log  = preg_replace( "/\[#(\d+)\]/", '[#$1](https://github.com/woocommerce/woocommerce/pull/$1)', $next_log );
-$changelog = preg_replace( "/^== Changelog ==\n\n/", "== Changelog ==\n\n{$next_log}\n", $changelog );
 
 $readme    = preg_replace( "/== Changelog ==\n(.*?)\[See changelog for all versions\]/s", "== Changelog ==\n\n{$next_log}\n\n[See changelog for all versions]", $readme );
 
 file_put_contents( $changelog_file, $changelog );
-file_put_contents( $readme_file, $readme );
```

Comment:
I think its the above line that should be removed?

Reply:
Good catch. It should've been the other one removed